tests/suites/hook_tools/state_tools.sh

```diff
@@ -0,0 +1,36 @@
+run_state_delete_get_set() {
+    echo
+
+    model_name="test-state-delete-get-set"
+    file="${TEST_DIR}/${model_name}.txt"
+
+    ensure "${model_name}" "${file}"
+
+    juju deploy cs:~jameinel/ubuntu-lite-7
+    wait_for "ubuntu-lite" "$(idle_condition "ubuntu-lite")"
+
+    juju run --unit ubuntu-lite/0 state-get | grep -q "{}"
+    juju run --unit ubuntu-lite/0 state-set one=two
```

Comment:
Don't you need to quote the juju run commands?
E.g.

```juju run --unit ubuntu-lite/0 'state-set one=two'```

Reply:
added